### ISSUE TITLE
Yaml Config help misses learning_rate_schedule property

### ISSUE THREAD

**Issue by JPhilipp** [Unity-Technologies/ml-agents#3328]
The [documentation of the Yaml config parameters](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Training-ML-Agents.md) is missing an explanatory entry for the optional `learning_rate_schedule` property, described more [here](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Training-PPO.md#optional-learning-rate-schedule).

(Sorry, not a bug, but didn't find a better fitting label.)

**Comment by anupambhatnagar** [comment]
thanks @JPhilipp for spotting the missing definition. We will update the docs accordingly.
